src/DwitTech.AccountService.Core/Services/ActivationService.cs

```diff
@@ -60,12 +56,55 @@ public bool SendActivationEmail(string fromEmail, string toEmail, string templat
             var baseUrl = GetBaseUrl();
             var activationUrl = GetActivationUrl();
             string templateText = GetTemplate(templateName);
-            templateText = templateText.Replace("{{name}}", RecipientName) ;
+            templateText = templateText.Replace("{{name}}", RecipientName);
             templateText = templateText.Replace("{{activationUrl}}", activationUrl);
             string body = templateText;
             var response = SendMail(fromEmail, toEmail, subject, body, cc, bcc);
 
             return response;
         }
+
+        public bool SendWelcomeEmail(string fromEmail, string toEmail, string templateName, string subject = "Account Details", string cc = "", string bcc = "")
+        {
+            var user = new User();
+            string templateText = GetTemplate(templateName);
+            templateText = templateText.Replace("{{Firstname}}", user.FirstName);
+            templateText = templateText.Replace("{{Lastname}}", user.LastName);
+            templateText = templateText.Replace("{{Email}}", user.Email);
+            templateText = templateText.Replace("{{Password}}", user.Password);
```

Comment:
I don't think this line should be here. We don't have the password to send

Reply:
> I don't think this line should be here. We don't have the password to send

Resolved